ckanext/xloader/loader.py

```diff
@@ -35,33 +37,69 @@
 SINGLE_BYTE_ENCODING = 'cp1252'
 
 
+class CanadaStream(Stream):
+
+    def __init__(self, source, *args, **kwargs):
+        super(CanadaStream, self).__init__(source, *args, **kwargs)
+        self.static_dialect = kwargs.get('static_dialect', None)
+        self.logger = kwargs.get('logger', None)
+
+    @property
+    def dialect(self):
+        """Dialect (if available)
+
+        # Returns
+            dict/None: dialect
+
+        """
+        if self.static_dialect:
+            if self.logger:
+                self.logger.info('Using Static Dialect for %s: %s', self.__format, dumps(self.static_dialect))
```

Comment:
This will fire on every file load. Should it be at DEBUG level instead of INFO?

Reply:
@ThrawnCA yeeeahhh haha, it will also fire off multiple times for a file possibly depending on the file encoding guessing thing.

I have yet to actually get this to work still. Just debugging it still to try and get the custom CSV parser to load. But does not seem to want to yet

Reply:
Limited logging in the `load_csv` and `load_table` methods. Keeping `info` log level for our fork as it is what we desire for our setup